pages/_document.js

```diff
@@ -33,6 +33,7 @@ export default class MyDocument extends Document {
     return (
       <Html>
         <Head>
+          <link rel="shortcut icon" href="/static/favicon.ico" />
```

Comment:
are you sure this link is not supposed to be in _app.js???

Reply:
Yeah I’m sure. This is how to add it